docs/peggy tutorial

```diff
@@ -0,0 +1,47 @@
+First install dependencies:
+
+```
+INFURA_PROJECT_ID="abcdebeedabcdebeed"
+ETHEREUM_PRIVATE_KEY="0x00000000000000000000000000000000000000"
+git clone https://github.com/Sifchain/sifnode
+cd sifnode/smart-contracts
+echo -e "ETHEREUM_PRIVATE_KEY=$ETHEREUM_PRIVATE_KEY\r\nINFURA_PROJECT_ID=$INFURA_PROJECT_ID" > .env
+curl -sL https://deb.nodesource.com/setup_14.x | sudo bash -
+sudo apt install nodejs
+sudo npm install -g truffle
+npm install dotenv
```

Comment:
Maybe point them to code that has all the dependencies (something like test/integration/setup-linux-environment.sh), or we have to commit to updating this document every time we add dependencies.

Reply:
Feel free to request changes